matlab/fiff_read_proj.m

```diff
@@ -45,6 +45,8 @@
 me='MNE:fiff_read_proj';
 
 if nargin ~= 2
+    rename_struct = struct();
+elseif nargin ~= 3
```

Comment:
This test sequence is weird. With nargin={0,1,4}, it passes.
The error 'Incorrect number of arguments' is never generated when needed.